src/deluge/gui/menu_item/formatted_title.h

```diff
@@ -10,7 +9,10 @@ class FormattedTitle {
 public:
 	FormattedTitle(l10n::String format_str) : format_str_(format_str) {}
 
-	void format(int32_t arg) { title_ = fmt::vformat(l10n::getView(format_str_), fmt::make_format_args(arg)); }
+	void format(int32_t arg) {
+		title_ = l10n::get(format_str_);
+		std::replace(title_.begin(), title_.end(), '*', (char)('0' + arg));
```

Comment:
Nit: The std::replace implementation takes 1696 bytes and if I read correctly does not support numbers > 9. Maybe we can replace it with something custom?

Reply:
Yeah fair enough - I'll go back to a custom one